src/main/java/com/trzewik/blackjack/Contestant.java

```diff
@@ -0,0 +1,34 @@
+package com.trzewik.blackjack;
+
+import java.util.ArrayList;
+
+public class Contestant {
+
+    ArrayList<Card> hand;
```

Comment:
```
    ArrayList<Card> hand;
    int position;
    boolean buster;
    int cash;
    String name;
```
powinno być prywatne - w JAVIE jest konwencja, że pola powinny być prywatne, a jak potrzebujesz mieć dostęp do tego pola, to robisz getter, np. getBuster(), getCash(). A jak chcesz ustawic jakas wartość, to robisz publicznego setera

Reply:
https://docs.oracle.com/javase/tutorial/java/javaOO/variables.html